core/src/main/java/utilcalc/core/parser/ParserUtils.java

```diff
@@ -0,0 +1,46 @@
+package utilcalc.core.parser;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.tomlj.TomlArray;
+import org.tomlj.TomlTable;
+
+public class ParserUtils {
+
+    static TomlTable requireNonNullTable(TomlTable table, String key) {
+        TomlTable value = table.getTable(key);
+        if (value == null) {
```

Comment:
Tento kus kodu se nam 4x opakuje, ale zatim si nejsem jisty, jestli by pomocna metoda pomohla nebo to spis zneprehlednila. Neco jako
```
static TomlTable requireNonNullTable(TomlTable table, String key) {
    return requireNonNull(table.getTable(key), key, "table");
}
```